.jshintrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Have you tried using eslint? We're moving towards using it at Facebook and would be awesome if this project would as well